git_data_model.sql

```diff
@@ -1,160 +1,337 @@
 CREATE TABLE users (
-user_id BIGSERIAL PRIMARY KEY NOT NULL,
-user_name VARCHAR (255) NOT NULL,
+id BIGSERIAL PRIMARY KEY NOT NULL,
+name VARCHAR (255) NOT NULL,
 email VARCHAR (255) NOT NULL,
 password VARCHAR(255) NOT NULL,
-UNIQUE (user_id)
+created_at TIMESTAMP WITHOUT TIME ZONE DEFAULT(NOW() AT TIME ZONE 'UTC'),
+UNIQUE (email)
 );
 
 
-INSERT INTO users (user_name, email, password)
+INSERT INTO users (name, email, password)
```

Comment:
you need to generate larger data set to see impact of indexes

Reply:
ok sir will generte it.